.github/workflows/ci.yml

```diff
@@ -1,19 +1,22 @@
 name: Continuous Integration
 
-on: [push, pull_request, workflow_dispatch]
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      trigger_docker:
+        description: "'parsec-service-test-all' if docker build should be triggered"
+        required: false
+        default: ""
 
 env:
-  # TEST_ALL_DOCKER_IMAGE: 'parsec-service-test-all'
-  TEST_ALL_DOCKER_IMAGE: 'ghcr.io/parallaxsecond/parsec-service-test-all'
+  TEST_ALL_DOCKER_IMAGE: ${{ github.event.inputs.trigger_docker || 'ghcr.io/parallaxsecond/parsec-service-test-all' }}
 
 jobs:
   build-and-export-test-all-docker:
     runs-on: ubuntu-latest
-    # If TEST_ALL_DOCKER_IMAGE is 'parsec-service-test-all' or any local image,
-    # the following condition must evaluate true to execute this job
-    # Else it must evaluate false to NOT execute this job
-    # Unfortunately, env.TEST_ALL_DOCKER_IMAGE cannot be used here as the `env` context is not recognized at this level.
-    if: ${{ false }} # env.TEST_ALL_DOCKER_IMAGE == 'parsec-service-test-all'
+    if: ${{ github.event.inputs.trigger_docker == 'parsec-service-test-all' }}
```

Comment:
Can you add a comment explaining how this job can be triggered?

Reply:
Please note that this is already written in the description of the trigger in line 9, which also shows up when manually triggering in the CI.

Reply:
Done